### ISSUE TITLE
Zooming on the graph causes the Quality Control State to update

### ISSUE THREAD

**Issue by arcturus2** [constellation-app/constellation#897]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Running the latest version of Constellation
	* Attached the ***Support Package*** via `Help` > `Support Package`
    * Checked the FAQs: https://github.com/constellation-app/constellation/wiki/FAQ
    * Checked that your issue isn't already filed: https://github.com/constellation-app/constellation/issues
    * Checked that there is not already a module that provides the described functionality: https://github.com/constellation-app/constellation/wiki/Catalogue-of-Repositories

### Description

Zooming the graph should not require the Quality Control State to update.

### Steps to Reproduce

1. Open a graph and the Quality Control View
2. Zoom in

**Expected behaviour:** [What you expect to happen]
The plugin reporter should not show anything

**Actual behaviour:** [What actually happens]
The Quality Control View State plugin runs.

**Reproduces how often:** [What percentage of the time does it reproduce?]
100%

### Additional Information
As part of this, please also make sure that the Quality Control State update is hidden in the Plugin Reporter and is only visible if the "Low Level" category is selected.

**Comment by aldebaran30701** [comment]
@arcturus2 or @antares1470 
I've just had a look at this issue, and it appears to be that when the global mod counters are different from the previous saved, it updates the state.

Moving forward, we have a few options.
A. Filter out updates like camera etc.
B. Add the attributes which the view is concerned about (not sure how exactly to scalably grab attributes from each rule to listen to)

I'd like to hear your thoughts, or suggestions on how to determine what events we update the state on.

**Comment by antares1470** [comment]
Just did some testing for this and noticed that the Quality Control state update wasn't popping up in the Plugin Reporter when doing a graph selection (this was with LOW LEVEL ticked in the filters). The issue appears to be the Plugin Info annotation for the `QualityControlViewStateUpdater`. I tried it out with the annotation removed and saw that the state update popped up when a selection was made but not when a zoom or other camera move was made (which is the intended behaviour). 

@arcturus2 @aldebaran30701 not sure how you want to approach this. We could either simply remove the Plugin Info annotation (which we know results in the intended behaviour) or we could look what needs to change to have both the annotation and QC state updates appearing in the plugin reporter.

**Comment by aldebaran30701** [comment]
@antares1470 I thought it was meant to show it as the annotation for low level. Not just get rid of it all together!
>As part of this, please also make sure that the Quality Control State update is hidden in the Plugin Reporter and is only visible if the "Low Level" category is selected.

I was attempting to recreate this which is seen on the original ticket.

**Comment by antares1470** [comment]
Right. So what I didn't notice in my testing was that LOW LEVEL was selected in the ignore category (though interestingly wasn't actually ticked so that might be a different bug). Once I removed that, I could see the expected changes (Side note: I actually managed to get a null pointer but it is apparently a known issue with the plugin reporter rather than your code so nothing to worry about there for this ticket).
